lua/gatekeeper/policylib.lua

```diff
@@ -281,3 +283,19 @@ function decision_web(policy, tx_rate_kib_sec, cap_expire_sec,
 		policy, tx_rate_kib_sec, tx_rate_kib_sec * 0.05, -- 5%
 		cap_expire_sec, next_renewal_ms, renewal_step_ms, false)
 end
+
+-- Estimate the number of rules and number of tbl8s in DPDK LPM library.
+function lpm_para_estimate(ipv4_file)
```

Comment:
This function must go to `lua/examples/policy.lua` because it depends on the policy; more specifically, on the format of the prefix file, or from wherever the prefixes are obtained.